commands/initial/templates/package.json

```diff
@@ -15,7 +15,9 @@
     "prebuild": "rimraf dist out-tsc",
     "start": "webpack-dev-server --open --config ./webpack/webpack.dev.js",
     "tagVersion": "np --no-publish",
-    "test": "node ./tasks/test"
+    "test": "node ./tasks/test",
+    "pree2e": "webdriver-manager update",
+    "e2e": "protractor"
```

Comment:
@gonzofish  A seperated script for e2e tests is always better. 
The reason lies within the fact that normally you would to do the following steps for a library (e.g. on a CI machine or during local development) and you wanna have a dedicated task for everyone of them:
 
1. Build: npm run build
1. Unit Test : npm run test
1. serve (local) resp. deploy (CI): npm run serve/deploy
1. e2e test: npm run e2e 

The last step should have at least the possibility to specify the selenium server used ( e.g.  localhost:4444 for local development and any other running selenium server when on CI). 

So it's always better to have two scripts/tasks for the two different types of test, due to their needed tasks that should in between (serve resp. deploy)